packages/elements-core/src/components/RequestSamples/convertRequestToSample.ts

```diff
@@ -10,3 +30,16 @@ export const convertRequestToSample = (language: string, library: string | undef
     return null;
   }
 };
+
+function toPartialHttpRequest(request: HarFormatRequest): PartialHttpRequest {
+  const query = request.queryString.reduce((obj, item) => Object.assign(obj, { [item.name]: [item.value] }), {});
```

Comment:
is that just copying objects? 🤔

Reply:
Almost. Both of these types are pretty similar e.g. 

<img width="479" alt="Screen Shot 2021-08-16 at 5 00 35 PM" src="https://user-images.githubusercontent.com/58433203/129584896-13af5376-87ab-4bd4-902b-e318edee8ef4.png">

vs

<img width="234" alt="Screen Shot 2021-08-16 at 5 19 08 PM" src="https://user-images.githubusercontent.com/58433203/129587540-e0134f41-ec6d-449f-b9de-2c4ad4800732.png">

Reply:
`PartialHttpRequest` interface which is the base of Stoplight Markdown syntax used in embedded `TryIt` is pretty loose. I guess we could add some guards here if needed